reproducible-build/build.sh

```diff
@@ -17,4 +17,4 @@ source setup
 
 make build || true
 
-sudo cp --dereference /build/srcs/poky/build/tmp/deploy/images/tdx/* /artifacts/.
+cp --dereference /build/srcs/poky/build/tmp/deploy/images/tdx/* /artifacts/.
```

Comment:
For this to work, you need to add your user to the docker's group like this [guide here](https://docs.docker.com/engine/install/linux-postinstall/).

The reason behind removing sudo was because upon finalizing the docker build, it is still prompting `enter password for poky to proceed` from within the docker build. So the artifacts were not being moved out and had to do that manually. 
This way, you just need to add the calling user to the docker group and it works directly without even using "sudo make azure-image"